### DISCUSSION TITLE
repeat attribute in animation not working in web

### DISCUSSION THREAD

**Discussion by jmatsushita** [tamagui/tamagui#3016] [General]
### Current Behavior

The [docs](https://github.com/tamagui/tamagui/blob/master/apps/site/data/docs/core/animations.mdx?plain=1#L250) imply that it's possible to use repeat for animations, however it doesn't seem to work. 

### Expected Behavior

`repeat` and other `moti` attributes like `loop` and `delay` are available in tamagui. This [discussion](https://github.com/tamagui/tamagui/discussions/553#discussioncomment-4731332) with @nandorojo seems to indicate that `loop` and `sequence` might have worked at some point? Hopefully, the new `devbox` setup should help demo simple tamagui recipes now https://github.com/tamagui/tamagui/issues/2075#issuecomment-2069527142

Alternatively it would be great to have some guidance on how to drop down one level to pass configuration down to moti?

### Tamagui Version

```markdown
1.94.5
```

### Platform (Web, iOS, Android)

```markdown
web
```

### Reproduction

https://codesandbox.io/p/devbox/angry-monad-t8rmdt?file=%2Fapp%2F%28tabs%29%2Findex.tsx

### System Info

```markdown
✗ npx envinfo --system --npmPackages --binaries --browsers

  System:
    OS: Linux 6.1 Debian GNU/Linux 12 (bookworm) 12 (bookworm)
    CPU: (2) x64 AMD EPYC
    Memory: 962.21 MB / 4.01 GB
    Container: Yes
    Shell: Unknown
  Binaries:
    Node: 20.12.0 - /usr/local/bin/node
    Yarn: 4.0.2 - /usr/local/bin/yarn
    npm: 10.5.0 - /usr/local/bin/npm
    pnpm: 8.15.6 - /usr/local/share/npm-global/bin/pnpm
  npmPackages:
    @babel/core: ^7.23.3 => 7.23.7 
    @expo/metro-runtime: ~3.1.3 => 3.1.3 
    @react-navigation/native: ^6.0.2 => 6.1.9 
    @tamagui/config: ^1.94.5 => 1.94.5 
    @tamagui/metro-plugin: ^1.94.5 => 1.94.5 
    @types/react: ~18.2.14 => 18.2.47 
    babel-preset-expo: ^10.0.1 => 10.0.1 
    expo: ~50.0.6 => 50.0.6 
    expo-font: ~11.10.2 => 11.10.2 
    expo-linking: ~6.2.1 => 6.2.2 
    expo-router: ~3.4.7 => 3.4.7 
    expo-splash-screen: ~0.26.4 => 0.26.4 
    expo-status-bar: ~1.11.1 => 1.11.1 
    expo-system-ui: ~2.9.3 => 2.9.3 
    expo-web-browser: ~12.8.2 => 12.8.2 
    react: ^18.2.0 => 18.2.0 
    react-dom: ^18.2.0 => 18.2.0 
    react-helmet-async: ~2.0.4 => 2.0.4 
    react-native: 0.73.4 => 0.73.4 
    react-native-reanimated: ~3.6.2 => 3.6.2 
    react-native-safe-area-context: 4.8.2 => 4.8.2 
    react-native-screens: ~3.29.0 => 3.29.0 
    react-native-web: ^0.19.10 => 0.19.10 
    tamagui: ^1.94.5 => 1.94.5 
    typescript: ^5.3.3 => 5.3.3
```

**Comment by nandorojo** [comment]
I think if you pass an array to the animate prop, the second item in the array will be the moti config
